GOVERNANCE.md

```diff
@@ -0,0 +1,193 @@
+* [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+* [Updates to this Document](#updates-to-this-document)
+* [Contact](#contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along the contributor ladder.  At all levels, contributors are required to follow the CNCF Code of Conduct (COC).
+
+Each of the project member roles below is organized into lists of three types of things.
+
+* "Responsibilities" – functions of a member
+* "Requirements" –  qualifications of a member
+* "Privileges" –  entitlements of member
+
+
+### Contributor
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the CNCF CoC
+    * Follow the project contributing guide
+* Requirements (one or several of the below):
+    * Report and sometimes resolve issues against any of the project’s repositories
+    * Occasionally submit PRs against any of the project’s repositories
+    * Contribute to project documentation, including the manpages, tutorials, and Podman.io
+    * Attend community meetings when reasonable
+    * Answer questions from other community members on the mailing list, Slack, Matrix, and other communication channels
+    * Assist in triaging issues
+    * Assist in reviewing pull requests, including testing patches when applicable
+    * Test release candidates and provide feedback
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become a Reviewer
+
+### Reviewer
+Description: A Reviewer has responsibility for the triage of issues and review of pull requests on the Podman project or a subproject, consisting of one or more of the Git repositories that form the project. They are collectively responsible, with other Reviewers, for reviewing changes to the repository or repositories and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Reviewers have all the rights and responsibilities of an Contributor, plus:
+
+* Responsibilities include:
+    * Regular contribution of pull requests to the Podman project or its subprojects
+    * Triage of Github issues on the Podman project or its subprojects
+    * Regularly fixing Github issues on the Podman project or its subprojects
+    * Following the reviewing guide
```

Comment:
Can we link to the guideline if there is one?

Reply:
Done